packages/component-base/src/virtualizer-iron-list-adapter.js

```diff
@@ -281,6 +281,19 @@ export class IronListAdapter {
       this._virtualCount = this.items.length;
     }
 
+    // When reducing size while invisible, iron-list does not update items, so
+    // their hidden state is not updated and their __lastUpdatedIndex is not
+    // reset. In that case do it manually here.
+    if (!this._visible) {
```

Comment:
This property `_visible` doesn't seem to exist so this always evaluates true

Reply:
Oops, meant to use `_isVisible`. Updated.